todoApp/spec/main.tsp

```diff
@@ -79,6 +79,11 @@ model TodoItem {
   @visibility("create") _dummy?: string;
 }
 
+model ToDoItemMultipartRequest {
+  item: HttpPart<TodoItem>;
+  attachments?: HttpPart<TodoAttachment[]>;
```

Comment:
@markcowl @lmazuel @bterlson Could you help confirm if we allow to share models among mfd and non-mfd ops: https://github.com/microsoft/typespec/issues/5376?

Also I created an issue to clarify the todo spec's scope: https://github.com/allenjzhang/typespec-e2e-demo/issues/34.

Reply:
I don't think we should share the model, it's quite unclear what an operation that takes a JSON in would understand to do with an `HttpPart` type. I think we discussed before that if `HttpPart` is used, then it's a MFD usage, and that's all.

Reply:
@lmazuel in todo spec the shared model is `ToDoItemMultipartRequest` but `TodoAttachment` in HttpPart.

Reply:
Personal opinion: TypeSpec `File` (or model extends from it) should not appear to JSON model.
It should only be used together with `HttpPart` as `HttpPart<File>`, at the property of a multipart payload.

It is possible that certain JSON model do have the same 3 properties, but I think in such case, dev better just write a model.